src/views/review-penalty.njk

```diff
@@ -26,14 +26,45 @@
             <h1 class="govuk-heading-xl">Penalty details for
               <span class="govuk-!-font-weight-bold">{{ companyName }}</span>
             </h1>
-            <h2 class="govuk-heading-m">Accounts period made up to {{ madeUpToDate }}</h2>
+            <h2 class="govuk-heading-m">Accounts period made up to {{ penalty.madeUpDate }}</h2>
```

Comment:
potentially look at changing madeUpDate to madeUpToDate?

Reply:
Thanks Lewis, Penalty object does not have madeUpToDate but [madeUpDate](https://github.com/companieshouse/ch-sdk-node/blob/6571ff2267a5ee9f93298d0abda62021c70407b4/src/services/lfp/types.ts#L11)